tests/functional/validation.py

```diff
@@ -108,6 +109,7 @@ def calculate_ss2_metrics_mtx(mtx_zip_url):
         shutil.copyfileobj(response.raw, local_mtx_zip_file)
 
     mtx_zip = zipfile.ZipFile(local_mtx_zip_path)
+    scanpy.read_10x_mtx(local_mtx_zip_path)
```

Comment:
TODO: clean up/refactor this functional test

Reply:
Unable to add functional tests until persistent 10x test bundles are available in dev environments.